src/connectors/kessel/impl.js

```diff
@@ -0,0 +1,199 @@
+'use strict';
```

Comment:
We'll want to add prometheus probes to this at some point but we can do that later...